src/util/get-files.ts

```diff
@@ -141,6 +141,7 @@ const asAbsolute = function(path: string, parent: string) {
 interface StaticFilesOptions {
   output: Output;
   isBuilds: boolean;
+  src?: string;
```

Comment:
Would be nice to have some comments for these args, or in the function comment for `staticFiles()` is good too.